run-checks

```diff
@@ -67,7 +67,7 @@ addtrap endmsg
 append_coverage() {
     local profile="$1"
     if [ -f $profile ]; then
-        cat $profile | grep -v "mode: set" >> .coverage/coverage.out
+        cat $profile | grep -v "mode: set" | cat >> .coverage/coverage.out
```

Comment:
Heh, if a package has 0% coverage (like the hooks package that doesn't really contain any code paths to test) grep exits non-zero and the whole script dies with no error, so adding `| cat` here avoids it.